generic_relations/relations.py

```diff
@@ -42,6 +42,9 @@ def field_to_native(self, obj, field_name):
         Delegates to the `to_native` method of the serializer registered
         under obj.__class__
         """
+        if not (obj and getattr(obj, field_name)):
```

Comment:
On second thought, this really is another issue and may not be warranted here.  i had another case where 'field_name' was a key in a dictionary, as the object returned was a dictionary.  In that case, i just made a subclass of this serializer that handled the different behavior in this method.  My vote is just to handle a `None` value for object here, and not worry about a `None` value for obj.field_name at least until another issue is raised independently.

How's that sound?  Anything else i will need to take care of before this can be accepted?